app/workers/pr_mergeability_checker.rb

```diff
@@ -27,20 +31,30 @@ def process_mergeability
       write_to_github
       apply_label
     elsif !was_mergeable && currently_mergeable
+      remove_comments
       remove_label
     end
 
     # Update columns directly to avoid collisions wrt the serialized column issue
     branch.update_columns(:mergeable => currently_mergeable)
   end
 
+  def remove_comments
+    GithubService.delete_comments(
+      fq_repo_name,
+      GithubService.issue_comments(fq_repo_name, branch.pr_number).select do |com|
+        com.user.login == Settings.github_credentials.username && com.body.eql?(unmergeable_comment)
+      end.map(&:id)
+    )
```

Comment:
Can you move out the block above to a variable so you don't have the weird `)` at the end?

Reply:
I think the usage of one variable at this point is useless. It will just help with the code readability.